.github/workflows/keep-workflows-enabled.yaml

```
@@ -0,0 +1,58 @@
+# This workflow is intended to keep scheduled GH Action workflows enabled
+# as a work-around for scheduled workflows being automatically disabled when
+# no repository activity has occurred in 60 days.
+name: Keep workflows enabled
+
+on:
+  schedule:
+    # Scheduled to run at 4pm UTC (8am PST) on the first day of the month
+    - cron: '0 16 1 * *'
+
+  workflow_dispatch:
+
+jobs:
+  enable-workflow:
+    permissions:
+      actions: write
+    env:
+      GH_TOKEN: ${{ secrets.GH_TOKEN_NEXTSTRAIN_BOT_WORKFLOW_DISPATCH }}
+    name: enable-workflow (${{ matrix.repo }}/${{ matrix.workflow }})
```

```suggestion
    name: "${{ matrix.repo }}:${{ matrix.workflow }}"
```

Removing the static prefix makes the workflow run page more useful. (Take or leave the other formatting change from `/` → `:`; I was just momentarily confused by the path-like name at first.)